docs/src/app/components/pages/components/BottomNavigation/ExampleSimple.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import FontIcon from 'material-ui/FontIcon';
+import {BottomNavigation, BottomNavigationItem} from 'material-ui/BottomNavigation';
+import Paper from 'material-ui/Paper';
+
+const recentsIcon = <FontIcon className="material-icons">restore</FontIcon>;
```

Comment:
Is this for readability?
Otherwise for performances, it's not needed with https://github.com/babel/babel/tree/master/packages/babel-plugin-transform-react-constant-elements.